layouts/jobs/bis.html

```diff
@@ -6,7 +6,8 @@
   {{ $header := dict
     "role" $role
     "job" .Parent.Params.job_name
-    "title" "Best-in-slot (BIS) Gear Sets"
+    "jobTitle" .Parent.Title
+    "title" "Best-in-slot (BiS) Gear Sets"
```

Comment:
One final nit – this can probably be `.Title`, since we have a title prop on all pages now (I should know, I did the [PR for it](https://github.com/The-Balance-FFXIV/balance-static/pull/725))